tests/tests-asn1c-compiler/00a-aper-tags-OK.asn1.-B

```diff
@@ -4,8 +4,8 @@
 
 syntax = "proto3";
 
-package pkg00a_aper_tags_ok_asn1.v1;
-option go_package = "pkg00a_aper_tags_ok_asn1/v1/moduletestconstraints1";
+package pkg00a_aper_tags_ok.v1;
+option go_package = "pkg00a_aper_tags_ok/v1/moduletestconstraints1;moduletestconstraints1";
```

Comment:
you're adding an alias at the end, but it's probably unnecessary since it's the same at the package name